app/models/repository.rb

```diff
@@ -282,7 +282,10 @@ def update_all_info_async(token = nil)
 
   def update_all_info(token = nil)
     token ||= AuthToken.token if host_type == "GitHub"
-    return unless check_status
+    unless check_status
+      update!(last_synced_at: Time.current)
+      return
+    end
```

Comment:
could consolidate this with the one below by moving them into an `ensure` section, e.g.:

```
  ...
  ensure
     update!(last_synced_at: Time.current)
  end
```

Reply:
hmm I don't think we want to use `ensure` here because we specifically aren't setting it unless we were able to gather all the data.

Reply:
oh, like if an exception was raised? True